File: MyCourses - Android/app/src/main/java/com/mycoursesapp/model/Course.java
package com.mycoursesapp.model;

import java.util.ArrayList;

/**
 * Created by mamdouhelnakeeb on 2/9/18.
 */

public class Course {

    public int id;
    public String name;
    public String slogan;
    public String img;
    public ArrayList<Category> categoryArrayList;

    public Course(int id, String name, String slogan, String img, ArrayList<Category> categoryArrayList){

        this.id = id;
        this.name = name;
        this.slogan = slogan;
        this.img = img;
        this.categoryArrayList = categoryArrayList;
    }

    public Course(){}
}

File: MyCourses - iOS/My Courses/Models/Course.swift
//
//  Course.swift
//  My Courses
//
//  Created by Mamdouh El Nakeeb on 3/11/18.
//  Copyright © 2018 My Courses. All rights reserved.
//

import Foundation

class Course {
    
    var id = 0
    var name = ""
    var slogan = ""
    var img = ""
    var categories = Array<Category>()
    
    init(id: Int, name: String, slogan: String, img: String){
        
        self.id = id
        self.name = name
        self.slogan = slogan
        self.img = img        
    }
    
    init() {}
}
